.github/workflows/build-operator-image.yaml

```diff
@@ -13,15 +13,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Image
-        run: |
-          IMG=eda-server-operator:main make docker-build
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
-      - name: Push To Quay
-        uses: redhat-actions/push-to-registry@v2.1.1
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
         with:
-          image: eda-server-operator
-          tags: main
-          registry: quay.io/ansible/
+          registry: quay.io
           username: ${{ secrets.QUAY_USER }}
           password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and Push Image
+        run: |
+          docker buildx build --push --platform=linux/arm64,linux/amd64,linux/s390x,linux/ppc64le --tag quay.io/ansible/eda-server-operator:main -f Dockerfile .
```

Comment:
reference #162 - https://github.com/ansible/eda-server-operator/pull/162/files#diff-32245d28b869933b2f68c1b883a196955a5315d1a80b36b645e4a897a522c6e8

The approach I took was to keep the same workflow of the original file, and add an additional step to store the build image on ghcr. This served the purpose of supporting future workflows, releases for instance. as now all that would need to be done for the release workflow is pull the ghcr image and publish it using the git tag.

The release workflow can also be viewed at #162 https://github.com/ansible/eda-server-operator/pull/162/files#diff-e380de8fb76960db94a9cf78262f3034b87215ed566a3d3db167d0b2125399bf.

I've also noticed #159 so will add comments there RE the changes from #158 that are beneficial and what I was doing in #162 .